packages/tests/esm/package.json

```diff
@@ -12,21 +12,26 @@
   "scripts": {
     "migrate": "DATABASE_URL=postgres://joist:local@localhost:5435/esm node-pg-migrate up --decamelize=false --migrations-dir=./migrations",
     "format": "prettier --ignore-path ../../../.prettierignore  --write '{schema,migrations,src}/**/*.{ts,js,tsx,jsx,graphql}'",
-    "codegen": "node ../../codegen",
+    "codegen": "node ../../codegen && graphql-codegen --config graphql-codegen.cjs",
     "test": "NODE_OPTIONS=--experimental-vm-modules jest --runInBand --logHeapUsage"
   },
   "dependencies": {
     "joist-orm": "workspace:*",
-    "node-pg-migrate": "^8.0.3"
+    "node-pg-migrate": "^8.0.3",
+    "temporal-polyfill": "^0.3.0"
   },
   "devDependencies": {
+    "@graphql-codegen/add": "^6.0.0",
+    "@graphql-codegen/cli": "^6.0.0",
+    "@homebound/graphql-typescript-simple-resolvers": "^1.57.0",
```

Comment:
Does it feel reasonable to have this in our test? Or do you want a separate package (`tests/esm-gql-codegen`) to test this logic?